lib/client/client.js

```diff
@@ -110,12 +110,23 @@ function Client (options) {
   EventEmitter.call(this, options)
 
   var self = this
-  var _url
-  if (options.url) { _url = url.parse(options.url) }
-  this.host = _url ? _url.hostname : undefined
-  this.port = _url ? _url.port : false
-  this.secure = _url ? _url.secure : false
-  this.url = _url
+  this.servers = []
```

Comment:
Just nitpicking but would `this.urls` be more appropriate? since it is a public property it would be better to get it right now so we don't need a major bump to change it later.